### PR TITLE
Yet another change to the gismo export

### PR DESCRIPTION
**Pull request by jzwar**
# Extend the export for user-options
There are some model-specific options, like assembler options or boundary conditions. Adding optional parameters to the export, helps avoid modifying the files manually once the simulation is runnng.
 

I am sorry that this is the 5th gismo related PR :

## Showcase
A short / one-liner example to highlight the (new) feature
```
import splinepy# Test Output against input
multipatch_geometry = splinepy.Multipatch([some, splines])

# Set some options
gismo_options = [
    {
        "tag": "Rostock",
        "attributes": {"Region": "MP", "population": "unfortunate"},
        "text": "",
        "children": [{"tag": "DM"}],
    }
]

splinepy.io.gismo.export(
  "fileName.xml",
  multipatch=multipatch_geometry,
  indent=False,
  labeled_boundaries=True,
  gismo_options=gismo_options
)

```

## Checklists
* [x] Documentations are up-to-date.
* [ ] Added example(s)
* [x] Added test(s)

### CODE REVIEW

File: splinepy/io/gismo.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ def export(
     multipatch=None,
     indent=True,
     labeled_boundaries=True,
+    gismo_options=None,
```
Comment (j042) [inline splinepy/io/gismo.py:15]:
```suggestion
    options=None,
```
seem redundant

File: splinepy/io/gismo.py [hunk 2]
```diff
@@ -305,18 +343,22 @@ def export(
         f.write(file_content)
 
 
-def load(fname):
+def load(fname, get_options=True):
```
Comment (j042) [inline splinepy/io/gismo.py:346]:
```suggestion
def load(fname, load_options=True):
```
do you need this? why not just always load options if it is specified?

File: splinepy/io/gismo.py [hunk 3]
```diff
@@ -305,18 +343,22 @@ def export(
...
     Parameters
     ----------
     fname : str
       filename of the gismo xml
+    get_options : bool
+      Retrieve additional options (else - multipatch/geometry only)
 
     Returns
     -------
     spline_dic_list : multipatch
```
Comment (j042) [inline splinepy/io/gismo.py:358]:
```suggestion
    multipatch: Multipatch
```

File: splinepy/io/gismo.py (hunk 2 above)
Comment (jzwar) [reply splinepy/io/gismo.py:346]: because it requires that a tuple is returned every time. If you are ok with that I can change that

**Pull request merged**